x-pack/plugins/ingest_manager/server/services/epm/packages/install.ts

```diff
@@ -203,13 +204,24 @@ export async function installPackage({
     type: ElasticsearchAssetType.indexTemplate,
   }));
   await Promise.all([installKibanaAssetsPromise, installIndexPatternPromise]);
+  const installedTransforms = await installTransformForDataset(
```

Comment:
I would think transforms should be installed before dashboards as they are more important for the ingestion path.

Reply:
I can move it up in the call.